Integrations/integration-TruSTAR.yml

```diff
@@ -910,4 +1092,191 @@ script:
     arguments: []
     description: Returns the list of all enclaves that the user has access to, as
       well as whether they can read, create, and update reports in that enclave.
+  - name: file
+    arguments:
+    - name: file
+      required: true
+      description: Enter file name, md5, sha1 or sha256
```

Comment:
It is not file name, but file hash. Also please use MD5, SHA-256 and SHA-1 instead of md5, sha256 and sha1